hw/mcu/nordic/nrf52xxx/syscfg.yml

```diff
@@ -449,6 +451,10 @@ syscfg.defs:
             - "!XTAL_RC"
         deprecated: 1
 
+syscfg.vals.MCU_NRF52810:
+    MCU_TARGET: nRF52810
+syscfg.vals.MCU_NRF52811:
+    MCU_TARGET: nRF52811
```

Comment:
vals below are for backwards compatibility with already existing syscfg definitions so this has no effect since there are no `MCU_NRF52810`/`MCU_NRF52811` defined

Reply:
Well that one didn't catch my eye. Fixed on https://github.com/apache/mynewt-core/pull/1856